[2023-06-15T21:52:13.953+0700] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: thethree_reqif_to_rst.upload_to_github manual__2023-06-15T14:51:39+00:00 [queued]>
[2023-06-15T21:52:13.960+0700] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: thethree_reqif_to_rst.upload_to_github manual__2023-06-15T14:51:39+00:00 [queued]>
[2023-06-15T21:52:13.961+0700] {taskinstance.py:1308} INFO - Starting attempt 1 of 0
[2023-06-15T21:52:13.976+0700] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): upload_to_github> on 2023-06-15 14:51:39+00:00
[2023-06-15T21:52:13.981+0700] {standard_task_runner.py:57} INFO - Started process 83577 to run task
[2023-06-15T21:52:13.988+0700] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'thethree_reqif_to_rst', 'upload_to_github', 'manual__2023-06-15T14:51:39+00:00', '--job-id', '162', '--raw', '--subdir', 'DAGS_FOLDER/mainflow.py', '--cfg-path', '/var/folders/v7/qsb_cvl92z92z6jf7g1jc0zr0000gn/T/tmp1anxfg93']
[2023-06-15T21:52:13.990+0700] {standard_task_runner.py:85} INFO - Job 162: Subtask upload_to_github
[2023-06-15T21:52:14.048+0700] {task_command.py:410} INFO - Running <TaskInstance: thethree_reqif_to_rst.upload_to_github manual__2023-06-15T14:51:39+00:00 [running]> on host macbook-pro.local
[2023-06-15T21:52:14.141+0700] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='The Three' AIRFLOW_CTX_DAG_ID='thethree_reqif_to_rst' AIRFLOW_CTX_TASK_ID='upload_to_github' AIRFLOW_CTX_EXECUTION_DATE='2023-06-15T14:51:39+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-15T14:51:39+00:00'
[2023-06-15T21:52:14.156+0700] {mainflow.py:48} INFO - {'AUTHENTICATION': {'TOKEN': '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', 'USERNAME': 'nxquang-al'}, 'MESSAGE': 'test-github-access-token', 'REPOSITORY': {'BRANCH': 'main', 'NAME': 'Test-github-api', 'OWNER': 'R2th'}}
[2023-06-15T21:52:14.156+0700] {mainflow.py:49} INFO - /Users/lap14202/.airflow/data/Requirements.rst
[2023-06-15T21:52:14.157+0700] {upload_to_github.py:21} INFO - https://api.github.com/repos/R2th/Test-github-api/contents/Users/lap14202/.airflow/data/Requirements.rst
[2023-06-15T21:52:14.157+0700] {upload_to_github.py:22} INFO - {'Authorization': 'Bearer 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', 'Content-Type': 'application/vnd.github+json'}
[2023-06-15T21:52:14.158+0700] {upload_to_github.py:23} INFO - main
[2023-06-15T21:53:59.580+0700] {local_task_job_runner.py:291} WARNING - State of this instance has been externally set to restarting. Terminating instance.
[2023-06-15T21:53:59.635+0700] {process_utils.py:131} INFO - Sending 15 to group 83577. PIDs of all processes in the group: [83577]
[2023-06-15T21:53:59.635+0700] {process_utils.py:86} INFO - Sending the signal 15 to group 83577
[2023-06-15T21:54:59.661+0700] {process_utils.py:149} WARNING - process psutil.Process(pid=83577, name='Python', status='running', started='21:52:13') did not respond to SIGTERM. Trying SIGKILL
[2023-06-15T21:54:59.662+0700] {process_utils.py:86} INFO - Sending the signal 9 to group 83577
[2023-06-15T21:54:59.670+0700] {process_utils.py:79} INFO - Process psutil.Process(pid=83577, name='Python', status='terminated', exitcode=<Negsignal.SIGKILL: -9>, started='21:52:13') (83577) terminated with exit code -9
[2023-06-15T21:54:59.671+0700] {standard_task_runner.py:172} ERROR - Job 162 was killed before it finished (likely due to running out of memory)
